internal/session/query.go

```diff
@@ -278,5 +278,100 @@ with
     where
       public_id in (select public_id from connections_to_close)
     returning public_id
+  `
+
+	// closeConnectionsForDeadServersCte finds connections that are:
+	//
+	// * not closed
+	// * belong to servers that have not reported in within an acceptable
+	// threshold of time
+	//
+	// and marks them as closed.
+	//
+	// The query returns the set of servers that have had connections closed
+	// along with their last update time and the number of connections closed on
+	// each.
+	closeConnectionsForDeadServersCte = `
+with
+  -- Get dead servers, parameterized off of grace period in seconds
+  dead_servers as (
+    select private_id, update_time
+    from server
+    where update_time < now() - interval '1 second' * $1
```

Comment:
Could also use the `wt_add_seconds_to_now` function here, I feel it reads better but as you like.  Could also create a new `wt_subtract_seconds_from_now` if you want to avoid sending the arg with a `-1*`